docs/src/content/docs/tr/reference/configuration.mdx

```diff
@@ -0,0 +1,457 @@
+		linkedin: 'https://www.linkedin.com/company/astroinc',
+		mastodon: 'https://m.webtoo.ls/@astro',
+		threads: 'https://www.threads.net/@nmoodev',
+		twitch: 'https://www.twitch.tv/bholmesdev',
+		twitter: 'https://twitter.com/astrodotbuild',
+		'x.com': 'https://x.com/astrodotbuild',
+		youtube: 'https://youtube.com/@astrodotbuild',
+	},
+});
+```
+
+### `customCss`
+
+**Tür:** `string[]`
+
+Starlight sitesinin görünümünü özelleştirmek için CSS dosyalarını sağlayın.
+
+Projektiğinizin kök dizinine göre yerel CSS dosyalarını destekler, örneğin `'./src/custom.css'`, ve npm modülü olarak yüklediğiniz CSS dosyalarını destekler, örneğin `'@fontsource/roboto'`.
+
+```js
+starlight({
+	customCss: ['./src/custom-styles.css', '@fontsource/roboto'],
+});
+```
+
+### `head`
+
+**Tür:** [`HeadConfig[]`](#headconfig)
+
+Starlight sitesinin `<head>` etiketine özel etiketler ekleyin.
+Analiz ve diğer üçüncü taraf betikleri ve kaynakları eklemek için kullanışlı olabilir.
+
+```js
+starlight({
+	head: [
+		// Örnek: Fathom analiz betik etiketi ekleyin.
+		{
+			etiket: 'betik',
+			attrs: {
+				src: 'https://cdn.usefathom.com/script.js',
+				'data-site': 'MY-FATHOM-ID',
+				defer: true,
+			},
+		},
+	],
+});
+```
+
+#### `HeadConfig`
+
+```ts
+interface HeadConfig {
+	tag: string;
+	attrs?: Record<string, string | boolean | undefined>;
+	content?: string;
+}
+```
+
+### `lastUpdated`
+
+**Tür:** `boolean`  
+**Varsayılan:** `false`
+
+Sayfanın en son ne zaman güncellendiğini kontrol edin.
+
+Varsayılan olarak, bu özellik, depo Git geçmişinize dayanır ve [yüzeysel klonlar](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt) gerçekleştiren bazı dağıtım platformlarında doğru olmayabilir. Bir sayfa bu ayarı veya Git tabanlı tarihi [`lastUpdated` sahasını](/reference/frontmatter/#lastupdated) kullanarak geçersiz kılabilir.
+
+### `pagination`
+
+**Tür:** `boolean`  
+**Varsayılan:** `true`
+
+Altbilginin önceki ve sonraki sayfa bağlantılarını içerip içermediğini tanımlar.
+
+Bir sayfa bu ayarı veya bağlantı metni ve/veya URL'sini [`prev`](/reference/frontmatter/#prev) ve [`next`](/reference/frontmatter/#next) önemli alanları kullanarak geçersiz kılabilir.
+
+### `favicon`
+
+**Tür:** `string`  
+**Varsayılan:** `'/favicon.svg'`
+
+Websitesi için varsayılan favicon'un yolunu belirler; bu favicon `public/` dizininde bulunmalıdır ve geçerli (`.ico`, `.gif`, `.jpg`, `.png`, veya `.svg`) bir simge dosyası olmalıdır..
+
+```js
+starlight({
+  favicon: '/images/favicon.svg',
+}),
+```
+
+Ek varyantlar veya yedek faviconlar belirtmeniz gerekiyorsa, bunları [`head` seçeneği](#head) kullanarak ekleyebilirsiniz:
+
+```js
+starlight({
+  favicon: '/images/favicon.svg'.
+  head: [
+    // Safari için ICO favicon yedekleme ekleyin.
+    {
+      etiket: 'link',
+      attrs: {
+        rel: 'ikon',
+        href:'/images/favicon.ico',
+        boyutlar: '32x32',
+      },
+    },
+  ],
+});
+```
+
+### `titleDelimiter | başlıkAyırıcı`
```

Comment:
In general we don’t translate these headings which are code options, so I guess this should match that?

```suggestion
### `titleDelimiter`
```